### PR TITLE
Devi/spi test tools

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for SpiTestTools app & added all the project files for windows 10 Iot Enterprise.

Following changes are done 

1) file modified : SpiTestTool.vcxproj
    Updated the Windows SDK version from 10.0.17763 to 10.0.19041
     Updated the Platform Toolset  from v141 to v142

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise

### CODE REVIEW

File: BusTools/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline BusTools/.gitkeep]: Unneeded .gitkeep file. Remove?

File: BusTools/SpiTestTool/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline BusTools/SpiTestTool/.gitkeep]: Unneeded .gitkeep file. Remove?

File: BusTools/.gitkeep (hunk 1 above)
Comment (v-dmudili) [reply BusTools/.gitkeep]: Removed...

File: BusTools/SpiTestTool/.gitkeep (hunk 2 above)
Comment (v-dmudili) [reply BusTools/SpiTestTool/.gitkeep]: Removed...

**Review by rsameser — APPROVED** [review]

**Review by neilsh-msft — APPROVED** [review]

**Pull request closed without merging**